src/actions/team.js

```diff
@@ -0,0 +1,23 @@
+import { Axios } from '../utils/axios';
+import localstorage from '../utils/localstorage';
+
+export const FETCHING_TEAM = 'FETCHING_TEAM';
+export const TEAM_FETCH_SUCCESS = 'TEAM_FETCH_SUCCESS';
+export const TEAM_FETCH_FAIL = 'TEAM_FETCH_FAIL';
+
+export const fetchTeamInfo = organization => async dispatch => {
+  dispatch({ type: FETCHING_TEAM });
+  try {
+    const { data } = await Axios(localstorage.get().token).get(
```

Comment:
Good Job with the Action Creator. Could you consider getting the token in the Axios Instance instead of passing it as an argument so as to make the Axios Instance more reusable and not getting local storage at each invocation?

Reply:
Resolved